ballerina/tests/resources/generated-sources/tool_test_external_imports/child_pb.bal

```diff
@@ -0,0 +1,272 @@
+import ballerina/grpc;
```

Comment:
Is licence block needed in test files? if so licence block is missing.

Reply:
These are test resource files that are used to asset the content of the bal files generated from the gRPC tool. Therefore they don't have license headers.